src/shape.cpp

```diff
@@ -363,21 +363,23 @@ std::size_t shape::index(std::size_t i) const
 
 std::vector<std::size_t> shape::multi(std::size_t idx) const
 {
-    if(idx >= elements())
-    {
-        MIGRAPHX_THROW("Can not convert element-index: " + std::to_string(idx) +
-                       " to multi-dimensional index for shape with num elements: " +
-                       std::to_string(elements()));
-    }
+    assert(idx < elements());
```

Comment:
The assert should probably go into `multi_copy` as well.

Reply:
Done.